diagnostics/model_test/verify_model.py

```diff
@@ -67,41 +67,72 @@ def calculate_output(param_dict, select_device, input_example):
         raise Exception("Input graph file '" + graph_filename +
                         "' does not exist!")
 
-    graph_def = tf.GraphDef()
-    if graph_filename.endswith("pbtxt"):
-        with open(graph_filename, "r") as f:
-            text_format.Merge(f.read(), graph_def)
-    else:
-        with open(graph_filename, "rb") as f:
-            graph_def.ParseFromString(f.read())
+    config = tf.ConfigProto(
+        allow_soft_placement=True,
```

Comment:
Why do you need soft placement=true?

Reply:
@avijit-nervana Hmm, it is not needed, will remove it. Thanks for pointing out.